Yona/Yona/Welcome/WelcomeViewController.swift

```diff
@@ -118,4 +81,32 @@ class WelcomeViewController: BaseViewController {
         }
     }
     
+    fileprivate func processEnvironmentURL(newEnvironmentURL:String, currentEnvironmentURL:String) {
+        EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
+        self.didEditTextField = false
+        if currentEnvironmentURL != newEnvironmentURL{
+            ActivitiesRequestManager.sharedInstance.getActivityDefaultCategories(onCompletion: { (success, message, code, activitiesReturned, error) in
+                if success { EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
+                }else{ EnvironmentManager.updateEnvironment(withURLString: currentEnvironmentURL)
+                }
+            })
+        }else{
```

Comment:
See my earlier comment about the code formatter. Spaces around the curly braces are missing (also on line 90).

Reply:
Done